js/package.json

```diff
@@ -53,53 +53,52 @@
     "npm-release.sh"
   ],
   "dependencies": {
-    "@types/flatbuffers": "1.9.0",
-    "@types/node": "10.5.1",
-    "@types/text-encoding-utf-8": "1.0.1",
-    "command-line-args": "5.0.2",
-    "command-line-usage": "5.0.5",
-    "flatbuffers": "1.9.0",
-    "json-bignum": "0.0.3",
-    "text-encoding-utf-8": "1.0.2",
-    "tslib": "1.9.3"
+    "@types/flatbuffers": "^1.9.0",
```

Comment:
@domoritz can we change these to exact versions instead of `^` minor updates?

Reply:
Sure. I thought it might make sense to change to updates since we now have a lock file. I happily do whatever you prefer.

Reply:
Yeah I agree with @trxcllnt, we should leave these as pinned dependencies for now. Perhaps there's a case to be made for making these minor updates but I think we should discuss that via mailing list or JIRA, rather than lumping it in with this PR.